src/peakstats.jl

```diff
@@ -28,7 +38,29 @@ function estimate_single_peak_stats(args...; calib_type::Symbol=:th228)
 end
 export estimate_single_peak_stats
 
-function _get_hist_peakpos_fwhm(E::AbstractVector, W::Vector{<:Real})
```

Comment:
Resolving the merge conflicts, something went wrong here.
`_get_hist_peakpos_fwhm` disappeared..